### ISSUE TITLE
How to set learning rate in training?

### ISSUE THREAD

**Issue by TsingWei** [Little-Podi/GRM#8]
It seems like the batch size is 40 on each GPU, and the learning rate is 0.0004 in total in the provided config.  How should I set the learning rate and the batch size if I have only one RTX3090 GPU?  BTW, could u please share ur training log?  I just wonder how to confirm when it is converged.

**Comment by Little-Podi** [comment]
I use 4 or 6 RTX 3090 GPUs for most of my experiments. I suggest setting the batch size to be as large as possible, so you can simply select the batch size by filling the capacity of your GPU memory. If you want to reproduce a similar training process, just make `num_gpus * batch_size * learning_rate` has the same result as the setting in my config. In fact, I think using the provided config also works fine. These settings won't make a significant difference unless the scales are changed by tens of times. I don't keep the training log at hand now. Sorry for that. I will save the training log next time. So many items to open-source nowadays.

**Comment by Clarkwk** [comment]
> num_gpus * batch_size * learning_rate

Hello, could you tell me where to set the three parameters you mentioned: num_gpus, batch_size and learning_rate, and get the same result as you? Is it set in the.yaml file, is it the BATCH_SIZE, NUM_WORKER and LR parameters in the yaml file?

**Comment by Little-Podi** [comment]
BATCH_SIZE and LR can be set in the yaml configs. For the number of GPUs, it is set [here](https://github.com/Little-Podi/GRM/blob/main/tracking/train.py#L22), you can customize it using [command](https://github.com/Little-Podi/GRM#training) when you launch the training.

**Comment by Clarkwk** [comment]
> BATCH_SIZE and LR can be set in the yaml configs. For the number of GPUs, it is set [here](https://github.com/Little-Podi/GRM/blob/main/tracking/train.py#L22), you can customize it using [command](https://github.com/Little-Podi/GRM#training) when you launch the training.

Thank you for your answer. Is this the same setting for a single GPU?

**Comment by Little-Podi** [comment]
For single GPU training, you can just ignore `--nproc` since the default "single" mode will not use this argument.
